source/hardware/controllers/numark_mixtrack_pro_3.rst

```diff
@@ -123,47 +123,29 @@ Adjusts the volume for headphone cueing in the software.
 ~~~~~~~~~~~~~~~~
 
 | Adjust High frequencies of the deck
-| **Shift + High :** Adjust FX 1, parameter 1 of the Effect Unit
-  assigned to the deck
-| **Padmode + high :** Adjust FX 2, parameter 1 of the Effect Unit
-  assigned to the deck
-| **Tap + high :** Adjust FX 2, parameter 1 of the Effect Unit assigned
-  to the deck
+| **Shift + High :** Adjust parameter 1 of the currently focused effect on this deck.
```

Comment:
In the future, we should switch to a tabular view (like the Pioneer DDJ-200 or Roland DJ-505 pages) and make use of the `:hwlabel:` directive, but I won't ask you to basically rewrite the whole manual page just for this bugfix :D

Reply:
Noted. I do like the look of the ddj manual. I'll have a look at a rewrite later. This will help with getting up to speed with git